recipes/zippy/all/conanfile.py

```diff
@@ -0,0 +1,31 @@
+from conans import ConanFile, tools
+import os
+
+class NoWideConan(ConanFile):
+    name = "zippy"
+    description = "A simple C++ wrapper around the \"miniz\" zip library "
+    topics = ("wrapper", "compression", "zip")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/troldal/Zippy"
+    license = "MIT"
+    exports_sources = "patches/*"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("miniz/2.2.0")
+
```

Comment:
You will need to add a `validate` method to check the `compiler.cppstd` and also exclude all compilers not supporting c++17.